CHANGELOG.md

```diff
@@ -45,6 +45,7 @@
 
 **Fixed**:
 
+- **decidim-ac countability**: Fix progress when provided. [\#3952](https://github.com/decidim/decidim/pull/3952)
```

Comment:
There's a :bug:'gy whitespace here :) Also, the text is a bit vague, better to say "Fix accountability progress to be between 0 and 100 if provided" like in the PR description!

Reply:
Thanks for your feedback !